### PR TITLE
Code refactoring of existence test cases

### PR DESCRIPTION
**Pull request by devendra104**

### CODE REVIEW

File: upgrade_tests/helpers/constants.py [hunk 1]
```diff
@@ -5,26 +5,23 @@
 
 # FAKE REPOS
 FAKE_REPO_ZOO3 = 'http://inecas.fedorapeople.org/fakerepos/zoo3/'
-to_version = os.environ.get('TO_VERSION')
+FROM_VERSION = os.environ.get('FROM_VERSION')
+TO_VERSION = os.environ.get('TO_VERSION')
+SAT_VERSION = ['6.2', '6.3', '6.4', '6.5', '6.6', '6.7']
```
Comment (jyejare) [inline upgrade_tests/helpers/constants.py:10]:
```suggestion
SUPPORTED_SAT_VERSIONS = ['6.2', '6.3', '6.4', '6.5', '6.6', '6.7']
```

File: upgrade_tests/helpers/existence.py [hunk 2]
```diff
@@ -375,17 +367,17 @@ def compare_postupgrade(component, attribute):
     predata = get_datastore('preupgrade', endpoint)
     postdata = get_datastore('postupgrade', endpoint)
     entity_values = []
-    atr = 'id' if endpoint == 'api' else cli_const.attribute_keys[component]
-    for test_case in find_datastore(predata, component, atr):
+    atr = 'id' if endpoint == 'api' else CLI_ATTRIBUTES_KEY[component]
+    for _ in find_datastore(predata, component, atr):
```
Comment (jyejare) [inline upgrade_tests/helpers/existence.py:371]: Any reason why `test_case` replaced with `_` ?

File: upgrade_tests/helpers/existence.py [hunk 3]
```diff
@@ -436,17 +428,15 @@ def compare_templates(template_type):
...
         raise IncorrectTemplateTypeException(
             'The Template Type has to be one of {}'.format(supported_templates))
-    from_ver = os.environ.get('FROM_VERSION')
-    to_ver = os.environ.get('TO_VERSION')
-    entity_values = []
-    for template_id in _find_templatestore('preupgrade', template_type):
-        prefile, pre_template = _find_templatestore('preupgrade', template_type, template_id)
-        postfile, post_template = _find_templatestore('postupgrade', template_type, template_id)
+    entity_values = list()
+    for _ in find_templatestore('preupgrade', template_type):
```
Comment (jyejare) [inline upgrade_tests/helpers/existence.py:432]: Why `_` ?

File: upgrade_tests/helpers/common.py [hunk 4]
```diff
@@ -108,4 +105,5 @@ def test_something():
     """
     versions = [versions] if type(versions) is str else versions
     return pytest.mark.skipif(
-        cur_ver in versions, reason='Not for version {}'.format(cur_ver))
+        f'{os.environ.get("FROM_VERSION")}' in versions,
```
[the remark this answers is not in the document]
Comment (devendra104) [reply upgrade_tests/helpers/common.py:108]: I will fix it in separate PR by using Dynaconf.

File: upgrade_tests/helpers/constants.py (hunk 1 above)
Comment (devendra104) [reply upgrade_tests/helpers/constants.py]: Fixed.

File: upgrade_tests/helpers/existence.py (hunk 2 above)
Comment (devendra104) [reply upgrade_tests/helpers/existence.py]: The intention was to reduce the unnecessary variable uses, but as per the offline discussion the variable name will help to understand the iterating item type.

File: upgrade_tests/helpers/existence.py (hunk 3 above)
Comment (devendra104) [reply upgrade_tests/helpers/existence.py]: fixed.

**Pull request merged**